test/view_component/slotable_v2_test.rb

```diff
@@ -398,4 +398,34 @@ def test_slot_type_collection_item?
   def test_slot_type_nil?
     assert_nil(SlotsV2Component.slot_type(:junk))
   end
+
+  def test_polymorphic_slot
```

Comment:
Can we add a test case for polymorphic `renders_one` slot that is (incorrectly) set twice?